ui/apps/platform/src/Containers/Policies/Detail/PolicyDetail.tsx

```diff
@@ -212,6 +234,17 @@ function PolicyDetail({
                                               >
                                                   Export policy to JSON
                                               </DropdownItem>,
+                                              isPolicyAsCodeEnabled ? (
+                                                  <DropdownItem
+                                                      key="Save as Custom Resource"
+                                                      component="button"
+                                                      onClick={onSaveAsCustomResource}
+                                                  >
+                                                      Save as Custom Resource
+                                                  </DropdownItem>
+                                              ) : (
+                                                  <> </>
```

Comment:
Today I learned. Does `Dropdown` element render the space? If yes, and for our learning, can you check with axe DevTools? Even if it reports an issue, we can let it slide.

Reply:
Also, for curiosity, does React report any error like `key` prop in Console pane?

Reply:
It does not render the space, and it does not result in any additional elements in the DOM.

It _does_ however report an error for this missing `key` prop -- good catch.